src/views/bootablevolumes/actions/components/EditBootableVolumesModal.tsx

```diff
@@ -157,24 +185,55 @@ const EditBootableVolumesModal: FC<EditBootableVolumesModalProps> = ({
                 </>
               }
             >
-              <FilterSelect
-                selected={instanceType}
-                setSelected={onInstanceTypeChange}
-                options={Object.keys(instanceTypesToSizesMap) || []}
-                groupVersionKind={VirtualMachineClusterInstancetypeModelGroupVersionKind}
-                optionLabelText={t('InstanceType')}
-              />
+              <Select
+                menuAppendTo="parent"
+                isOpen={isInstanceTypeOpen}
+                onToggle={setIsInstanceTypeOpen}
+                onSelect={onInstanceTypeSelect}
+                variant={SelectVariant.single}
+                placeholderText={t('Select InstanceType')}
+                selections={instanceType}
+              >
+                {Object.keys(InstanceTypeCategory)?.map((instanceTypeCategory) => {
+                  const { seriesLabel, title }: CategoryDetails =
+                    categoryDetailsMap[instanceTypeCategory];
+                  return (
+                    <SelectOption
+                      key={instanceTypeCategory}
+                      value={instanceTypeCategory}
+                      description={title}
+                    >
+                      {seriesLabel}
+                    </SelectOption>
+                  );
+                })}
+              </Select>
             </FormGroup>
           </GridItem>
           <GridItem span={6}>
             <FormGroup label={t('Size')}>
-              <FilterSelect
-                selected={size}
-                setSelected={setSize}
-                options={sizeOptions}
-                groupVersionKind={VirtualMachineClusterInstancetypeModelGroupVersionKind}
-                optionLabelText={t('size')}
-              />
+              <Select
+                menuAppendTo="parent"
```

Comment:
please check if this doesn't cause any bugs, such as the dropdown sometimes opening up instead of down, anyway, I'm not sure u need this line

Reply:
Great just make sure that it ain't open up on some resolution on chrome and firefox. I encounter this bug with menu append

Reply:
Tested in various browsers - Firefox, Chrome, Chromium, different resolutions and size of the browser window, works as expected.

Reply:
Tested in various browsers - Firefox, Chrome, Chromium, different resolutions and size of the browser window, works as expected.